### PR TITLE
Introducing Economics. Update the API of the promises. Replace mana/gas with balance. Expand balance API.

### PR DESCRIPTION
**Pull request by nearmax**
3 part change:
https://github.com/nearprotocol/nearcore/pull/952
https://github.com/nearprotocol/near-runtime-ts/pull/20
https://github.com/nearprotocol/nearlib/pull/14

See the roadmap for Economics Implementation and the list of changes in these 3 PRs here:
https://docs.google.com/document/d/1rydPgSHwPaKQLL57Xogsw8tgjTPKG98cK3kgK_PnjCQ/edit#

### CODE REVIEW

File: near.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ class Near {
         options.changeMethods.forEach((methodName) => {
             contract[methodName] = async function (args) {
                 args = args || {};
-                const response = await near.scheduleFunctionCall(0, options.sender, contractAccountId, methodName, args);
+                const response = await near.scheduleFunctionCall(1000000000, options.sender, contractAccountId, methodName, args);
```
Comment (janedegtiareva) [inline near.js]: what does this number mean? could you pull this out into const with a comment?

File: nearclient.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ class NearClient {
         const buffer = SignedTransaction.encode(signedTransaction).finish();
         const transaction = _arrayBufferToBase64(buffer);
         const params = [transaction];
-        const response = await this.jsonRpcRequest('broadcast_tx_async', params);
+        const response = await this.jsonRpcRequest('broadcast_tx_sync', params);
```
Comment (janedegtiareva) [inline nearclient.js]: are tests passing with this?

Comment (nearmax) [reply nearclient.js]: Yes

File: near.js (hunk 1 above)
Comment (nearmax) [reply near.js]: Added constant and comment.

File: dist/nearlib.js [hunk 3]
```diff
@@ -350,6 +350,8 @@ const {
 
 const MAX_STATUS_POLL_ATTEMPTS = 10;
 const STATUS_POLL_PERIOD_MS = 2000;
+// Default amount to be send with the function calls.
```
Comment (janedegtiareva) [inline dist/nearlib.js]: what does this "amount" mean? what's going to happen to it?

Comment (nearmax) [reply dist/nearlib.js]: Added more explanation.

**Pull request merged**